### ISSUE TITLE
put multiple notes/subtrees into one workspace

### ISSUE THREAD

**Issue by andrewchenshx** [zadam/trilium#3607]
is it possible to put multiple notes/subtrees into one workspace?

**Comment by zadam** [comment]
Maybe I don't understand the question, but sure. Any note (including workspace) can have children, subtrees.

**Comment by andrewchenshx** [comment]
my bad.
say the root note has following subnote
:
Trilium Demo
work1
work2
work3
…

that's work1,work2 and work3 are direct child notes of root.
then I would like to put work1,work2 and work3 into a workspace named current_work.
is it possible?

thanks

**Comment by zadam** [comment]
Yes, you can put workspaces arbitrarily deep into the note hierarchy and also create workspaces within workspaces.
